train.py

```diff
@@ -338,19 +344,118 @@ def evaluate_model_vs_random(model: torch.nn.Module, epsilon: float, num_games:
     win_rate = player_wins / num_games
     draw_rate = draws / num_games
 
-    return win_rate, draw_rate
+    return -win_rate, draw_rate
+
+
+class CheckersQModelWrapper(BaseEstimator, RegressorMixin):
```

Comment:
Let's create a new file for the added classes and functions related to hparam search